packages/components/src/icons/Copy.stories.tsx

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
@CornelMaze we have a component called <Icon />

The copy icon used is feather icon called copy. So this file and story is not required